### PR TITLE
docs: cleanup readme

### PR DESCRIPTION
**Pull request by rishabh3112**
**What kind of change does this PR introduce?**
Documentation change

**Did you add tests for your changes?**
N/A

**If relevant, did you update the documentation?**
Yes

**Summary**
This readme is visible to the user of `npm` and places other than github.
Thus it doesn't align with readme(s) of other official webpack projects.
There are some sections with TODO in readme which should not be there in release.

**Does this PR introduce a breaking change?**
No

**Other information**
https://www.npmjs.com/package/webpack-cli

### CODE REVIEW

**Review by snitin315 — APPROVED** [review]

File: packages/webpack-cli/README.md [hunk 1]
```diff
@@ -1,4 +1,15 @@
...
+        <img width="200" height="200" src="https://webpack.js.org/assets/icon-square-big.svg">
+    </a>
+</div>
+
+<h1 align="center">webpack CLI</h1>
+
+<p align="center">
+  The official CLI of webpack
+</p>
+<br>
```
Comment (evilebottnawi) [inline packages/webpack-cli/README.md:12]: Let's prefer to use markdown in markdown, like:
```md
<div align="center">
    <a href="https://github.com/webpack/webpack-cli">
        <img width="200" height="200" src="https://webpack.js.org/assets/icon-square-big.svg">
    </a>
</div>

# webpack CLI

The official CLI of webpack
```

Comment (rishabh3112) [reply packages/webpack-cli/README.md]: done

**Review by evilebottnawi — APPROVED** [review]

**Review by snitin315 — APPROVED** [review]

**Review by jamesgeorge007 — APPROVED** [review]

File: packages/webpack-cli/README.md [hunk 2]
```diff
@@ -1,4 +1,12 @@
-# `webpack-cli`
+<div align="center">
+    <a href="https://github.com/webpack/webpack-cli">
+        <img width="200" height="200" src="https://webpack.js.org/assets/icon-square-big.svg">
+    </a>
+</div>
+
+# webpack CLI
```
Comment (anshumanv) [inline packages/webpack-cli/README.md:7]: Let's center these text too, looks very odd

File: packages/webpack-cli/README.md (hunk 1 above)
Comment (anshumanv) [reply packages/webpack-cli/README.md]: I'll suggest keeping HTML to center the headings and description, can't center with MD

File: packages/webpack-cli/README.md (hunk 2 above)
Comment (rishabh3112) [reply packages/webpack-cli/README.md:7]: https://github.com/webpack/webpack-cli/pull/2035#discussion_r517286159

File: packages/webpack-cli/README.md (hunk 1 above)
Comment (evilebottnawi) [reply packages/webpack-cli/README.md]: It is documentation, not web page, so you read this from left to right (in most of languages), no need centered it (especial on very large monitors )

Comment (rishabh3112) [reply packages/webpack-cli/README.md]: NPM doesn't center the headings anyway (https://www.npmjs.com/package/webpack) so no need.
Also, dev server does same as well (https://github.com/webpack/webpack-dev-server/blob/master/README.md).

Comment (anshumanv) [reply packages/webpack-cli/README.md]: seems fair

**Review by anshumanv — APPROVED** [review]

File: packages/webpack-cli/README.md (hunk 2 above)
Comment (anshumanv) [reply packages/webpack-cli/README.md:7]: https://github.com/webpack/webpack-cli/pull/2035#discussion_r517504804

**Review by evenstensberg — APPROVED** [review]

**Pull request merged**